libs/application-generic/src/usecases/validate-control-by-tier/validate-control-by-tier.command.ts

```diff
@@ -0,0 +1,16 @@
+import { StepTypeEnum } from '@novu/shared/';
+import { IsEnum, IsNumber, IsOptional } from 'class-validator';
+
+import { OrganizationLevelCommand } from '../../commands';
+
+export type Milliseconds = number;
+
+export class ValidateControlByTierCommand extends OrganizationLevelCommand {
+  @IsNumber()
+  @IsOptional()
+  deferDuration?: Milliseconds;
```

Comment:
As above `defer` is a confusing new term. I would simplify to "maximum"
```suggestion
  maximumDuration?: Milliseconds;
```

Reply:
Let's continue the talk about the name on this [comment](https://github.com/novuhq/novu/pull/7088#discussion_r1850485669) 

I loved the Ms suffix, i will add it :)